// @google-cloud/dialogflow/build/src/v2beta1/index.js
"use strict";
// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
// ** This file is automatically generated by gapic-generator-typescript. **
// ** https://github.com/googleapis/gapic-generator-typescript **
// ** All changes to this file may be overwritten. **
Object.defineProperty(exports, "__esModule", { value: true });
var agents_client_1 = require("./agents_client");
Object.defineProperty(exports, "AgentsClient", { enumerable: true, get: function () { return agents_client_1.AgentsClient; } });
var contexts_client_1 = require("./contexts_client");
Object.defineProperty(exports, "ContextsClient", { enumerable: true, get: function () { return contexts_client_1.ContextsClient; } });
var documents_client_1 = require("./documents_client");
Object.defineProperty(exports, "DocumentsClient", { enumerable: true, get: function () { return documents_client_1.DocumentsClient; } });
var entity_types_client_1 = require("./entity_types_client");
Object.defineProperty(exports, "EntityTypesClient", { enumerable: true, get: function () { return entity_types_client_1.EntityTypesClient; } });
var environments_client_1 = require("./environments_client");
Object.defineProperty(exports, "EnvironmentsClient", { enumerable: true, get: function () { return environments_client_1.EnvironmentsClient; } });
var intents_client_1 = require("./intents_client");
Object.defineProperty(exports, "IntentsClient", { enumerable: true, get: function () { return intents_client_1.IntentsClient; } });
var knowledge_bases_client_1 = require("./knowledge_bases_client");
Object.defineProperty(exports, "KnowledgeBasesClient", { enumerable: true, get: function () { return knowledge_bases_client_1.KnowledgeBasesClient; } });
var session_entity_types_client_1 = require("./session_entity_types_client");
Object.defineProperty(exports, "SessionEntityTypesClient", { enumerable: true, get: function () { return session_entity_types_client_1.SessionEntityTypesClient; } });
var sessions_client_1 = require("./sessions_client");
Object.defineProperty(exports, "SessionsClient", { enumerable: true, get: function () { return sessions_client_1.SessionsClient; } });
