// @google-cloud/dialogflow/build/src/index.js
"use strict";
// Copyright 2020 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
//
// ** This file is automatically generated by synthtool. **
// ** https://github.com/googleapis/synthtool **
// ** All changes to this file may be overwritten. **
Object.defineProperty(exports, "__esModule", { value: true });
exports.protos = exports.SessionsClient = exports.SessionEntityTypesClient = exports.IntentsClient = exports.EnvironmentsClient = exports.EntityTypesClient = exports.ContextsClient = exports.AgentsClient = exports.v2 = exports.v2beta1 = void 0;
const v2beta1 = require("./v2beta1");
exports.v2beta1 = v2beta1;
const v2 = require("./v2");
exports.v2 = v2;
const AgentsClient = v2.AgentsClient;
exports.AgentsClient = AgentsClient;
const ContextsClient = v2.ContextsClient;
exports.ContextsClient = ContextsClient;
const EntityTypesClient = v2.EntityTypesClient;
exports.EntityTypesClient = EntityTypesClient;
const EnvironmentsClient = v2.EnvironmentsClient;
exports.EnvironmentsClient = EnvironmentsClient;
const IntentsClient = v2.IntentsClient;
exports.IntentsClient = IntentsClient;
const SessionEntityTypesClient = v2.SessionEntityTypesClient;
exports.SessionEntityTypesClient = SessionEntityTypesClient;
const SessionsClient = v2.SessionsClient;
exports.SessionsClient = SessionsClient;
exports.default = {
    v2beta1,
    v2,
    AgentsClient,
    ContextsClient,
    EntityTypesClient,
    EnvironmentsClient,
    IntentsClient,
    SessionEntityTypesClient,
    SessionsClient,
};
const protos = require("../protos/protos");
exports.protos = protos;
